client/components/typing-box.jsx

```diff
@@ -15,13 +16,51 @@ const TypeText = styled('div')({
   margin: '0.5rem'
 });
 
+const BlurBox = styled('div')({
+  minWidth: '100%',
+  minHeight: '100%',
+  backdropFilter: 'blur(4px)',
+  WebkitBackdropFilter: 'blur(4px)',
+  position: 'absolute',
+  pointerEvents: 'none'
+});
+
+const BlurMessage = styled('div')({
+  position: 'absolute',
+  transform: 'translate(-50%, -50%)',
+  top: '50%',
+  left: '50%',
+  fontFamily: 'retro, sans-serif',
+  color: 'white'
+});
```

Comment:
Can we move this to below the box and change the color to red?

Reply:
![](https://i.imgur.com/aqXKHtK.png)

I changed the color to red, but think it looks pretty good in the center. Instead, I increased the blur effect to make the instruction text stand out more.

Reply:
![](https://i.imgur.com/fueXMRp.png)

Increased font size slightly as well